app/controllers/katello/application_controller.rb

```diff
@@ -49,7 +50,7 @@ def section_id
       'generic'
     end
 
-    def current_organization
+    def current_organization_data
```

Comment:
optional nitpick: `s/_data/_object/` to be more descriptive

Reply:
Do you have a reasoning for this? I generally try to not name variables by what data structure they are so I went with the more general `data`. I'm open to changing though

Reply:
It's a low-information word that doesn't convey much in a name IMO, like 'helper' or 'manager' (not that 'object' is much better). It is a small gripe and I don't have a good alternate suggestion, I won't block on it. I'll change my approval to :heavy_check_mark: once https://github.com/Katello/katello/pull/7874#discussion_r240223930 is investigated.

Reply:
I like _object better, but both are bad.  Foreman's helper ideally would have been named _name and thus we wouldn't even have this problem, as current_organization is the best name for it :)

Reply:
ya, it does seem strange `current_organization` just returns the name, but naming stuff is hard :man_shrugging: I can change to `_object` since you both are in favor of that